File: 0001 Code Chef/0013 Thala For a Reason/implementation/0010 nodejs/thala_for_reason.js
const readline = require('readline');

const rl = readline.createInterface({
    input:process.stdin,
    output:process.stdout
});


rl.question('',(input)=>{
    number = parseInt(input);
    if(number == 7){
        console.log("THALA");
    }else{
        console.log("SADGE");
    }
    rl.close();
})

File: 0001 Code Chef/0013 Thala For a Reason/implementation/0005 kotlin/ThalaForReason.kt
fun main(args:Array<String>){
    var stringInput:String? = readLine();
    if(stringInput != null){
        var number:Int? = stringInput.toIntOrNull();
        if(number != null){
            if(number == 7){
                println("THALA");
            }else{
                println("SADGE");
            }
        }
    }
}